src/graphic-device/dot-use-examples/index.md

```diff
@@ -31,10 +31,12 @@ The app splash screen utilises the dot in motion to represent GOV.UK bringing to
 </div>
 {% endgrid %}
 
-### Video description
+### Video description of long and short splash screens
 
 No audio. The dot leads a trail of other dots of various colours in a spiral towards the centre of a mobile screen, where it becomes the dot within the GOV.UK logo as it fades and pushes in. The crown logo is revealed by a circular iris effect at the bottom of the screen.
 
+In the long version, the trail of dots moves in a wider spiral.
```

Comment:
I would be interested to hear from @mia-allers-gds if it's okay to have two paragraphs here or if we should try to keep everything in a single paragraph.